highfidelity/hifi
# Replacing jump_takeoff.fbx and jump_land.fbx animations to fix transition to fly bug

This pr fixes bug with tpose in the transition to fly.
The bug is here: https://highfidelity.manuscript.com/f/cases/17588/Avatar-goes-to-T-pose-while-trying-to-fly-when-walking-running

### Test plan:  
1) open interface and start walking and then press the pg up key to fly.
You should no longer see a t-pose happening in the transition to and from flying
2) repeat 1 with running and flying.
